packages/main/src/plugin/index.ts

```diff
@@ -1024,6 +1024,10 @@ export class PluginSystem {
       return telemetry.sendFeedback(feedbackProperties);
     });
 
+    this.ipcHandle('telemetry:pageOpened', async (_listener, url: string): Promise<void> => {
+      return telemetry.track('pageOpened', { url: url });
```

Comment:
@slemeur do you expect as event data the generic URL or some page name ?

Reply:
It would be more helpful with the page name.

Reply:
Also, I would expect we use the event "page" instead of track:
https://segment.com/docs/connections/sources/catalog/libraries/server/http-api/#page